docs/packetcapture-guide.md

```diff
@@ -0,0 +1,89 @@
+# Packet Capture User Guide
+
+Starting with Antrea v2.2, Antrea supports PacketCapture for network diagnosis.
+It can capture specified number of packets from real traffic and upload them to a
+supported storage location. Users can create a `PacketCapture` CR to trigger
+packet capture on the target traffic flow.
+
+## Prerequisites
+
+PacketCapture is disabled by default. If you
+want to enable this feature, you need to set feature gate `PacketCapture` to `true` in
+the `antrea-config` ConfigMap for `antrea-agent`.
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: antrea-config
+  namespace: kube-system
+data:
+  antrea-agent.conf: |
+    featureGates:
+      PacketCapture: true
+```
+
+## Start a new PacketCapture
+
+When starting a new packet capture, you can provide the following information to identify
+the target traffic flow:
+
+* Source Pod
+* Destination Pod, Service or IP address
```

Comment:
```suggestion
* Destination Pod, or destination IP address
```

IIRC we removed Service support for now?

Reply:
Yes.